examples/scala-sbt-client-usage/build.sbt

```diff
@@ -2,12 +2,12 @@ name := "scala-sbt-client-usage"
 
 version := "0.1"
 
-scalaVersion := "2.12.10"
+scalaVersion := "2.13.6"
```

Comment:
When we migrated to 2.13, the example project was still on 2.12 and was consuming a cached version of the client library. This only became a problem after bumping the global version.